### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by rukundoeric**
## What does this do?

- weather app

## Description of Task to be completed?
In this project, I built a simple weather app
 
## How should this be manually tested?
- Clone this repo https://github.com/rukundoeric/weather-app
- Navigate to the project directory.
- If you have liveserver in your vs-code, just run it.
- Otherwise, just open the `dist/index.html` page with your browser.
OR
- Simply Open the livedemo [Here](https://raw.githack.com/rukundoeric/weather-app/weather-app/dist/index.html)

### CODE REVIEW

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @rukundoeric 

Great job. Your code is straight forward, clean and webpack is equally well configured.
code2

Project works well too
code

### [changes required]

- we need to toggle between the temperature units [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)]
- There should be a background image change (design) based on the returned value [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png)]
- Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts`. 
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

On top of your hard work on this project, I think you need a little bit more courage to add these things to meet up with the requirements.
Waiting for a final review, hopefully.
### Happy coding

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @rukundoeric, congratulations on your hard work :100: Having fixed the comments outlined in the previous review :+1: Now we can toggle the units, the background changes based on the data received, you also added the "dev" and "build" scripts :smile: Awesome

I hereby approve the project, kindly go ahead and merge it into development. Do not forget to submit the project completion form :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**
